src/main/webapp/WEB-INF/tags/instructor/feedbackEdit/questionGeneralSettings.tag

```diff
@@ -0,0 +1,227 @@
+<%@ tag description="instructorFeedbacks - feedback question settings common to all types" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%@ tag import="teammates.common.util.Const" %>
+<%@ tag import="teammates.common.util.FieldValidator" %>
+<%@ tag import="teammates.logic.core.Emails.EmailType" %>
+<%@ tag import="teammates.common.datatransfer.FeedbackParticipantType" %>
+
+<%@ attribute name="fqForm" type="teammates.ui.template.FeedbackQuestionEditForm" required="true"%>
+<div class="col-sm-12 padding-15px margin-bottom-15px background-color-light-green">
+    <div class="col-sm-12 padding-0">
+        <b>Feedback Path</b> (Who is giving feedback about whom?)
+    </div>
+    <div class="col-sm-6 padding-0"
+        data-toggle="tooltip" data-placement="top"
+        title="<%= Const.Tooltips.FEEDBACK_SESSION_GIVER %>">  
+        <label class="col-sm-5 control-label">
+            Who will give the feedback:
+        </label>
+        <div class="col-sm-7">
+            <select class="form-control participantSelect"
+                id="<%= Const.ParamsNames.FEEDBACK_QUESTION_GIVERTYPE %>${fqForm.questionNumberSuffix}"
+                name="<%= Const.ParamsNames.FEEDBACK_QUESTION_GIVERTYPE %>"
+                <c:if test="!fqForm.isEditable">disabled="disabled"</c:if>
+                onchange="feedbackGiverUpdateVisibilityOptions(this)">
+                <c:forEach items="${fqForm.generalSettings.giverParticipantOptions}" var="option">
+                    <option <c:forEach items="${option.attributes}" var="attr"> ${attr.key}="${attr.value}"</c:forEach> >
+                        ${option.content}
+                    </option>
+                </c:forEach>
+            </select>
+        </div>
+    </div>
+    <div class="col-sm-6 padding-0" data-toggle="tooltip" data-placement="top"
+        title="<%= Const.Tooltips.FEEDBACK_SESSION_RECIPIENT %>">
+        <label class="col-sm-5 control-label">
+            Who the feedback is about:
+        </label>
+        <div class="col-sm-7">
+            <select class="form-control participantSelect"
+                id="<%= Const.ParamsNames.FEEDBACK_QUESTION_RECIPIENTTYPE %>${fqForm.questionNumberSuffix}"
+                name="<%= Const.ParamsNames.FEEDBACK_QUESTION_RECIPIENTTYPE %>"
+                <c:if test="!fqForm.isEditable">disabled="disabled"</c:if> onchange="feedbackRecipientUpdateVisibilityOptions(this);getVisibilityMessageIfPreviewIsActive(this);">
```

Comment:
Same here. (`${}` in `c:if`'s test)
Check the rest of your `c:if`s, it happens a lot more times.